app.py

```diff
@@ -1,37 +1,47 @@
-from flask import Flask, request, render_template_string, redirect, url_for
+from flask import Flask, request, render_template, redirect, url_for
 import json
 import os 
 app= Flask(__name__)
-DATA_FILE = "mooduri.json"
+DATA_FILE = "moods.json"
 
-if os.path.exists(DATA_FILE):
-    with open(DATA_FILE, "r") as f:
-        moods = json.load(f)
-else:
-    moods = []
+moods = []
+
+def load_moods():
+    if os.path.exists(DATA_FILE):
+        with open(DATA_FILE, "r") as f:
+            try:
+                return json.load(f)
+            except json.JSONDecodeError:
+                return []
+    return[]
```

Comment:
```suggestion
    return []
```

Greseala de sintaxa, testeaza dupa fiecare modificare